tox.ini

```diff
@@ -46,6 +46,7 @@ commands =
   coverage3 run --source circuit_knitting_toolbox --parallel-mode -m pytest test/ {posargs}
   coverage3 combine
   coverage3 html
+  coverage3 report --fail-under=100 --show-missing --include="circuit_knitting_toolbox/circuit_cutting/wire_cutting/wire_cutting_verification.py,circuit_knitting_toolbox/utils/integral_driver.py"
```

Comment:
I was going to approve this, but I think we can remove integral_driver since it's being removed from the repo

Reply:
I removed it in 39d197624cc205eb80cd178bd27a9e643b69573d, but we should remember that when we _do_ want to add additional file(s) or path(s), they should be separated by commas.